### ISSUE TITLE
acf_is_screen throws error in ajax requests

### ISSUE THREAD

**Issue by Syberspace** [elliotcondon/acf#525]
The api helper function `acf_is_screen` does not correctly handle the return values of wordpress' `get_current_screen`

Code from api/api-helpers.php:
```php
function acf_is_screen( $id = '' ) {	
	// vars
	$current_screen = get_current_screen();
	
	// return
	return ($id === $current_screen->id);	
}
```
As described in the [Codex](https://codex.wordpress.org/Function_Reference/get_current_screen), `get_current_screen` can return null in some cases.

I found this issue because I have set up a filter on acf/load_field that checks 
`if( acf_is_screen('acf-field-group') )` and when I call `get_field` on any ajax call I get a php notice that destroyes the ajax response.

**Comment by elliotcondon** [comment]
Hi @Syberspace 

I'm happy to add in a null check into this function, but please keep in mind this is a private function that is not intended to be used in any other place than after the current_screen action.

**Comment by ControlledChaos** [comment]
It is still breaking bbPress, even with the addition of `if( !$current_screen ) return false;`

**Comment by ControlledChaos** [comment]
The error occurs running the Twenty Sixteen theme 1.2 (current) and only ACF 5.4.0 (current) and bbPress 2.5.10 (current) plugins activated, no template mods on bbPress.

No error if I replace `$current_screen = get_current_screen():` with `global $current_screen;`

**Comment by elliotcondon** [comment]
Hi @ControlledChaos 

Thanks for the replied. I'll do some testing with this today and get back to you

**Comment by ControlledChaos** [comment]
Cool, a one line fix! Working as of v5.4.1.
